README.rst

```diff
@@ -354,7 +354,8 @@ and the `2-Clause BSD License <bsd-license_>`__
 .. _`BPF and XDP Reference Guide`: https://docs.cilium.io/en/stable/bpf/
 .. _`eBPF`: https://ebpf.io
 .. _`eBPF.io`: https://ebpf.io
-.. _`Join Zoom`: https://zoom.us/j/596609673
+.. _`Meeting Notes and Zoom Info`: https://docs.google.com/document/d/1Y_4chDk4rznD6UgXPlPvn3Dc7l-ZutGajUv1eF0VDwQ/edit#
+.. _`APAC Meeting Notes and Zoom Info`: https://docs.google.com/document/d/1egv4qLydr0geP-GjQexYKm4tz3_tHy-LCBjVQcXcT5M/edit#
```

Comment:
@YutaroHayakawa @youngnick can we add the Zoom meeting link in the google doc?

Reply:
Sorry, I don't get what you mean. We do have a zoom link in the google doc already.

Reply:
Zoom meeting link is already present in the google doc. The link that was here before was an older/deprecated one. Instead of replacing just the zoom link, I linked the google doc here, which not only has the correct/new zoom link but other relevant info and agenda.

Reply:
> I'm sorry I might be missing something. I don't see any zoom link in this doc https://docs.google.com/document/d/1egv4qLydr0geP-GjQexYKm4tz3_tHy-LCBjVQcXcT5M/edit#

Hmm @aanm, I see the following on the very top of the doc:

![Screenshot 2024-10-24 at 5 14 48 PM](https://github.com/user-attachments/assets/672783a7-fca0-4e19-8086-1b01328dc2c5)

Do you mean something else?

Reply:
> > I'm sorry I might be missing something. I don't see any zoom link in this doc https://docs.google.com/document/d/1egv4qLydr0geP-GjQexYKm4tz3_tHy-LCBjVQcXcT5M/edit#
> 
> Hmm @aanm, I see the following on the very top of the doc:
> 
> ![Screenshot 2024-10-24 at 5 14 48 PM](https://private-user-images.githubusercontent.com/1638019/379938995-767e4d72-03b6-4bd8-99ff-ebb165198697.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjk4MDU3MTMsIm5iZiI6MTcyOTgwNTQxMywicGF0aCI6Ii8xNjM4MDE5LzM3OTkzODk5NS03NjdlNGQ3Mi0wM2I2LTRiZDgtOTlmZi1lYmIxNjUxOTg2OTcucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTAyNCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDEwMjRUMjEzMDEzWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9OTZhZWQ3NGRhMGEwYjYwZjRiYmNjMWRiNDY2NzkwMDZkMmI1ZDE2M2I5MjkxYmExZDdjMThlOTNlMmM0YWNhNiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.nAVewBiJ5LxaYNRLiSMSVpdcjGbZLtDjljl_-26ieCM)
> 
> Do you not see this or do you mean something else?


I saw some strangeness in the doc where the excerpt about the meeting info was being marked as the 'First page header' and hence was not single-clickable. I tried to fix it, hopefully it is a bit better?

Reply:
I can see the link now and it's clickable.

Reply:
Now I see the link. I didn't see it before. Thanks everyone!